indexer/src/main/java/au/org/aodn/esindexer/service/AodnDiscoveryParameterVocabService.java

```diff
@@ -0,0 +1,70 @@
+package au.org.aodn.esindexer.service;
+
+import au.org.aodn.esindexer.abstracts.OgcApiRequestEntityCreator;
+import au.org.aodn.esindexer.utils.CacheArdcVocabsUtils;
+import au.org.aodn.stac.model.ConceptModel;
+import au.org.aodn.stac.model.ThemesModel;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Service
+public class AodnDiscoveryParameterVocabService {
+    @Autowired
```

Comment:
Which one is using?

Reply:
you meant the `RestTemplate` bean? The indexer uses `RestTemplate` bean configured at: https://github.com/aodn/es-indexer/blob/48810c61b84a60942d5034738cda90e9d4e2d285/indexer/src/main/java/au/org/aodn/esindexer/configuration/WebMvcConfig.java#L49

The indexer doesn't use stuff configured for the ARDC related code that you created, when I moved them into this repo, I treat them as a black box.  But both using `RestTemplate`, so I had to change the bean name of the ARDC lib, and use the `@Qualifier`.

Reply:
anyway, I have made a few changes to bean name and stuff, some was renamed, some was deleted.